tests/ha/ha_cluster_init.pm

```diff
@@ -53,7 +55,12 @@ sub run {
     exec_csync;
 
     # State of SBD if shared storage SBD is used
-    assert_script_run "sbd -d $sbd_device list" unless (get_var('USE_DISKLESS_SBD'));
+    if (!get_var('USE_DISKLESS_SBD')) {
+        $num_nodes  = get_node_number;
+        $sbd_output = script_output("sbd -d $sbd_device list");
```

Comment:
As `sbd_output` is used only in the `if` section, I think that you could set it here (`my $sbd_output = script_...`).